addons/event/data/mail_template_data.xml

```diff
@@ -99,16 +101,23 @@
                                 <img src="/web_editor/font_to_img/61555/rgb(81,81,102)/34" style="padding:4px;max-width:inherit;" height="34" alt=""/>
                             </td>
                             <td style="padding: 0px 10px 0px 10px;width:50%;line-height:20px;vertical-align:top;">
-                                <div>
-                                    <strong>From</strong>
-                                    <t t-out="object.event_id.date_begin" t-options='{"widget": "datetime", "date_only": True, "tz_name": object.event_id.date_tz, "format": "long"}'>May 4, 2021</t>
-                                     - <t t-out="object.event_id.date_begin" t-options='{"widget": "datetime", "time_only": True, "tz_name": object.event_id.date_tz, "hide_seconds": True, "format": "short"}'>7:00 AM</t>
-                                </div>
-                                <div>
-                                    <strong>To</strong>
-                                    <t t-out="object.event_id.date_end" t-options='{"widget": "datetime", "date_only": True, "tz_name": object.event_id.date_tz, "format": "long"}'>May 6, 2021</t>
-                                     - <t t-out="object.event_id.date_end" t-options='{"widget": "datetime", "time_only": True, "tz_name": object.event_id.date_tz, "hide_seconds": True, "format": "short"}'>5:00 PM</t>
-                                </div>
+                                <t t-if="object.event_id.is_one_day">
+                                    <div>
+                                        <t t-out="object.event_id.date_begin" t-options='{"widget": "datetime", "date_only": True, "tz_name": object.event_id.date_tz, "format": "long"}'>May 4, 2021</t>
+                                        <t t-out="object.event_id.date_begin" t-options='{"widget": "datetime", "time_only": True, "tz_name": object.event_id.date_tz, "hide_seconds": True, "format": "short"}'>7:00 AM</t>
+                                         - <t t-out="object.event_id.date_end" t-options='{"widget": "datetime", "time_only": True, "tz_name": object.event_id.date_tz, "hide_seconds": True, "format": "short"}'>5:00 PM</t>
+                                    </div>
+                                </t>
+                                <t t-else="">
+                                    <div>
+                                        <strong>From</strong>
+                                        <t t-out="object.event_id.date_begin" t-options='{"widget": "datetime", "show_seconds": False, "tz_name": object.event_id.date_tz, "format": "medium"}'>May 4, 2021 - 7:00 AM</t>
+                                    </div>
+                                    <div>
```

Comment:
Can't you use a single div (to nest with t-else) + br ? That's a bit how we create content now (div + br yay).